tests/vmi_multus_test.go

```diff
@@ -658,8 +658,11 @@ var _ = Describe("[Serial]SRIOV", func() {
 
 		waitVmi := func(vmi *v1.VirtualMachineInstance) {
 			// Need to wait for cloud init to finish and start the agent inside the vmi.
-			tests.WaitAgentConnected(virtClient, vmi)
+			vmi, err := virtClient.VirtualMachineInstance(vmi.Namespace).Get(vmi.Name, &metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
```

Comment:
As we have only explicit expect in this function (rest are on the invoked ones)
do we want WithOfffest 1 ?
as there are around 7 caller to waitVmi